packages/itwinui-react/src/core/Carousel/CarouselSlider.tsx

```diff
@@ -86,14 +87,13 @@ export const CarouselSlider = React.forwardRef((props, ref) => {
 
   return (
     <Box
-      as='ol'
-      aria-live='polite'
```

Comment:
i believe live region is unnecessary here because we do not support (or recommend) auto-rotating carousels. since the slide changes are triggered manually, the user doesn't need to be "notified" (which is the purpose of a live region).